### PR TITLE
No height overflow if we have info box

### PR DESCRIPTION
**Pull request by peregrineshahin**
This makes the right panel accept as many features one can add there; by making it scrollable instead of the height overflowing leaving empty space when extra widgets are added.

See:
Capture
vs after scroll:
Capture2

### CODE REVIEW

File: server/fishtest/templates/tests_view.mak [hunk 1]
```diff
@@ -4,6 +4,11 @@
   import json
 %>
 
+<%
```
Comment (ppigazzini) [inline server/fishtest/templates/tests_view.mak:7]: Independent from template arguments, move in this tag.
```
<%!
```

**Comment by vondele** [comment]
don't we have now two parallel scroll bars? That's usually a bit painful, IMO. 

BTW, personally, I really would like to have the diff next to the 'approve' button, i.e. above the details. It's good to have the diff visible without scrolling, I think.

**Comment by peregrineshahin** [comment]
> don't we have now two parallel scroll bars? That's usually a bit painful, IMO.
> 
> BTW, personally, I really would like to have the diff next to the 'approve' button, i.e. above the details. It's good to have the diff visible without scrolling, I think.

as the diffs are fetched from external server rendering on initial load would make the page bumpy if we move it above, I have another idea since we have 3 different places we go to the github diff url I think it is best if we make the `diff` for each test in the home page when clicked instead of going to github it can scroll for you to the diff box when clicked? what do you think?

File: server/fishtest/templates/tests_view.mak [hunk 2]
```diff
@@ -72,6 +77,7 @@
           <thead></thead>
           <tbody>
             % for arg in run_args:
+              % args_row_count += 1
```
Comment (ppigazzini) [inline server/fishtest/templates/tests_view.mak:80]: replace everywhere:
```diff
- % args_row_count += 1
+ <% args_row_count += 1 %>
```

Comment (ppigazzini) [reply server/fishtest/templates/tests_view.mak:80]: DEV updated with the fixes.

**Comment by ppigazzini** [comment]
DEV updated

File: server/fishtest/templates/tests_view.mak (hunk 1 above)
Comment (ppigazzini) [reply server/fishtest/templates/tests_view.mak]: Ooops, it seems that I was wrong, please revert this change, it seems that in that way the names become const. 

```yaml
May 27 15:45:29 dfts-0 pserve[27882]:   File "/home/usr00/fishtest/server/fishtest/templates/tests_view.mak", line 80, in render_body
May 27 15:45:29 dfts-0 pserve[27882]:     <% args_row_count += 1 %>
May 27 15:45:29 dfts-0 pserve[27882]: UnboundLocalError: cannot access local variable 'args_row_count' where it is not associated with a value
```

File: server/fishtest/templates/tests_view.mak [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 <%!
   import json
+  args_row_count = 0
+  args_row_height = "33px"
```
Comment (ppigazzini) [inline server/fishtest/templates/tests_view.mak:6]: Ooops, it seems that I was wrong, please revert this change, it seems that in that way the names become const. 

```yaml
May 27 15:45:29 dfts-0 pserve[27882]:   File "/home/usr00/fishtest/server/fishtest/templates/tests_view.mak", line 80, in render_body
May 27 15:45:29 dfts-0 pserve[27882]:     <% args_row_count += 1 %>
May 27 15:45:29 dfts-0 pserve[27882]: UnboundLocalError: cannot access local variable 'args_row_count' where it is not associated with a value
```

**Comment by peregrineshahin** [comment]
> Ooops, it seems that I was wrong, please revert this change, it seems that this way the names become const.

Fixed.

**Comment by peregrineshahin** [comment]
Superseded by #1709 .
